### PR TITLE
Add autonomous commands

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

**Review by ErikBoesen — CHANGES_REQUESTED** [review]
Conflicts.

**Comment by AndrewLester** [comment]
Yeah I'm going to try to fix these from my own computer.

File: src/main/java/frc/robot/commands/AlignCommand.java [hunk 1]
```diff
@@ -11,9 +11,9 @@
 
     private static final double Ki = 0.00200;
     private static final double Kd = 0.000004;
-    private static double Kp = 0.017000;
+    private static double Kp = 0.01700;
 
-    protected final PIDController pid = new PIDController(Kp, Ki, Kd);
+    protected PIDController pid = new PIDController(Kp, Ki, Kd);
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/AlignCommand.java:16]: why not keep this as final and just call `setP`, etc

Comment (AndrewLester) [reply src/main/java/frc/robot/commands/AlignCommand.java:16]: Good point. Wait one second, I'm going to try to push a few edits for this branch after rebasing it with the main one.

**Comment by AndrewLester** [comment]
Closed in favor of #55

**Pull request closed without merging**